File: packages/graph/__fixtures__/extractor-project/Enums.java
public enum OrderStatus {
    PENDING,
    CONFIRMED,
    SHIPPED,
    DELIVERED,
    CANCELLED
}

public enum Priority {
    LOW,
    MEDIUM,
    HIGH,
    CRITICAL
}

public enum PaymentMethod {
    CREDIT_CARD("credit_card"),
    DEBIT_CARD("debit_card"),
    PAYPAL("paypal"),
    BANK_TRANSFER("bank_transfer");

    private final String value;

    PaymentMethod(String value) {
        this.value = value;
    }
}

File: packages/graph/__fixtures__/extractor-project/enums.rs
pub enum OrderStatus {
    Pending,
    Confirmed,
    Shipped,
    Delivered,
    Cancelled,
}

pub enum Priority {
    Low,
    Medium,
    High,
    Critical,
}

pub enum PaymentMethod {
    CreditCard,
    DebitCard,
    PayPal,
    BankTransfer,
}

pub enum ShippingType {
    Standard(u32),
    Express(u32),
    Overnight,
}
